### PR TITLE
Add `lowercase` option

### PR DESCRIPTION
**Pull request by lagden**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -35,20 +35,27 @@ module.exports = (string, options) => {
 
 	options = Object.assign({
 		separator: '-',
-		customReplacements: []
+		customReplacements: [],
+		lowerCase: true
 	}, options);
 
 	const separator = escapeStringRegexp(options.separator);
+	const {lowerCase} = options;
```
Comment (sindresorhus) [inline index.js]: Really no benefit to destructure here. Just use `if (options.lowerCase) {`

File: index.js [hunk 2]
```diff
@@ -35,20 +35,27 @@ module.exports = (string, options) => {
...
 	const customReplacements = new Map([
 		...builtinReplacements,
 		...options.customReplacements
 	]);
 
+	let regex = /[^a-zA-Z\d]+/g;
```
Comment (sindresorhus) [inline index.js]: `regex` is too generic. It should have a more specific name.

File: test.js [hunk 3]
```diff
@@ -52,3 +52,12 @@ test('custom replacements', t => {
 		]
 	}), 'i-amour-licorne');
 });
+
+test('custom lowercase', t => {
```
Comment (sindresorhus) [inline test.js]: > lowerCase option

File: index.js [hunk 4]
```diff
@@ -35,20 +35,27 @@ module.exports = (string, options) => {
 
 	options = Object.assign({
 		separator: '-',
-		customReplacements: []
+		customReplacements: [],
+		lowerCase: true
```
Comment (sindresorhus) [inline index.js]: The option should be named `lowercase`.

See: https://english.stackexchange.com/questions/59409/lowercase-lower-case-or-lower-case

**Comment by sindresorhus** [comment]
The option needs to be documented in the readme.

**Comment by lagden** [comment]
Okay, I'll make the corrections soon
Tks!

**Comment by sindresorhus** [comment]
Looks good now. Thanks :)

**Pull request merged**